unity-renderer/Assets/Scripts/MainScripts/DCL/AvatarSystem/BaseAvatar.cs

```diff
@@ -77,16 +77,17 @@ UniTask GetRevealTask(Material material, float revealPosition, float completionP
                                .ToUniTaskInstantCancelation(true, cancellationToken: linkedCts.Token);
             }
 
-            baseAvatarReferences.ParticlesContainer.SetActive(true);
-            List<UniTask> tasks = new List<UniTask>();
-            tasks.Add(GetRevealTask(ghostMaterial, avatarHeight, completionHeight));
-
-            for (var index = 0; index < cachedMaterials.Count; index++)
+            try
             {
-                tasks.Add(GetRevealTask(cachedMaterials[index], -avatarHeight, -completionHeight));
-            }
+                baseAvatarReferences.ParticlesContainer.SetActive(true);
+                List<UniTask> tasks = new List<UniTask>();
+                tasks.Add(GetRevealTask(ghostMaterial, avatarHeight, completionHeight));
+
+                for (var index = 0; index < cachedMaterials.Count; index++) { tasks.Add(GetRevealTask(cachedMaterials[index], -avatarHeight, -completionHeight)); }
```

Comment:
I would suggest 2-lines split for readability
```
for (var index = 0; index < cachedMaterials.Count; index++)
    tasks.Add(GetRevealTask(cachedMaterials[index], -avatarHeight, -completionHeight));
```

Reply:
Done, the EditorConfig we have forced that, I disabled locally my auto-cleanup and will do it manually =D